src/core/client/ui/components/v2/Icon/Icon.css

```diff
@@ -0,0 +1,59 @@
+.root {
+  composes: icon from "coral-ui/shared/icon.css";
```

Comment:
do we want this to be referencing styles outside of coral-admin?

Reply:
I missed this, thank you for catching it, will get it updated. :)